services/QuillLMS/client/app/bundles/Staff/components/comprehension/rulesAnalysis/ruleAnalysis.tsx

```diff
@@ -184,11 +184,16 @@ const RuleAnalysis = ({ history, match }) => {
     { name: "", attribute:"value", width: "750px" }
   ];
 
+  const extractHighlight = (highlightObject) => {
+    if (!highlightObject || !highlightObject.length || highlightObject.length < 1 || !highlightObject[0].text) return '';
+    return highlightObject[0].text;
+  }
```

Comment:
Since this function doesn't use any props or state, we can define it outside of the component function, which will make it easier to pull into a shared file later if we want to use it elsewhere.